source/examples/rapids-autoscaling-multi-tenant-kubernetes/notebook.ipynb

```diff
@@ -110,35 +87,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 45,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "daemonset.apps/nvidia-driver-installer created\n"
```

Comment:
It looks like all of the cell outputs have been removed from this notebook. That's a bit of a problem because it will remove all of the results and graphs from the rendered page. Could you put these back? It probably easier to just grab a fresh copy of the original notebook and remove the `DISABLE_JUPYTER` but and then save it.